media-api/app/lib/elasticsearch/SearchFilters.scala

```diff
@@ -90,4 +79,9 @@ trait SearchFilters extends ImageFields {
     "pool"
   )
 
+  def filterOrFilter(filter: Option[FilterBuilder], orFilter: Option[FilterBuilder]): Option[FilterBuilder] = (filter, orFilter) match {
```

Comment:
:+1: I swear there has to be some Scalaz incantation to do this in a clever, succinct, type-agnostic way.

@bmjames would know...